ddd-repository-spring-data/src/test/java/eu/socialedge/ddd/infrastructure/persistence/jpa/SpringExpandableRepositoryTest.java

```diff
@@ -0,0 +1,90 @@
+package eu.socialedge.ddd.infrastructure.persistence.jpa;
+
+import eu.socialedge.ddd.domain.repository.RepositoryException;
+import eu.socialedge.ddd.infrastructure.persistence.jpa.domain.Ent1ty;
+import eu.socialedge.ddd.infrastructure.persistence.jpa.domain.repository.Ent1tyExpandableRepository;
+import lombok.val;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.transaction.annotation.Transactional;
+
+import static eu.socialedge.ddd.infrastructure.persistence.jpa.domain.util.Ent1ties.randomEnt1ties;
+import static eu.socialedge.ddd.infrastructure.persistence.jpa.domain.util.Ent1ties.randomEnt1ty;
+import static org.junit.Assert.*;
+
+@RunWith(SpringRunner.class)
+@DataJpaTest
+@SpringBootTest(classes = SpringRepositoryTestConfig.class)
+public class SpringExpandableRepositoryTest {
+
+    @Autowired
+    private TestEntityManager entityManager;
+
+    @Autowired
+    private Ent1tyExpandableRepository entityRepository;
+
+    @Before
+    public void setUp() {
+        entityManager.clear();
+        entityManager.flush();
+    }
+
+    @Test
+    public void shouldAddEnt1tyCorrectly() {
+        val randEntity = randomEnt1ty();
+        val randEntityId = randEntity.id();
+
+        entityRepository.add(randEntity);
+
+        assertFalse(entityRepository.isEmpty());
```

Comment:
This check seems redundand (size() != 0L), because in next line you check (size() == 1l) that will work just fine on its own